src/runtime_src/core/tools/xbutil2/XBReport.cpp

```diff
@@ -31,6 +31,15 @@ using namespace XBReport;
 
 
 // ------ F U N C T I O N S ---------------------------------------------------
+void
+report_driver_version(boost::property_tree::ptree _pt, std::string pt_var, std::string msg_var)
+{
+  if (_pt.get<std::string>(pt_var, "NA") != std::string("NA")) { 
```

Comment:
The previous author of the code used "N/A" and here "NA" is being used.   Did you really intend to change the default return value to something else.  If so, what is the reasoning behind it.